### ISSUE TITLE
Yoroi is not syncing - ledger wallet 

### ISSUE THREAD

**Issue by redoracle** [Emurgo/yoroi-frontend#2579]
After many attempts also by removing and re-pairing the ledger hw wallet, Yoroi keeps syncing and after hours and even days the wallet does not sync.

(only the rewards are loaded, everything else related to the wallet is not)

**Comment by benapetr** [comment]
yes having the same problem you are probably hitting this https://github.com/Emurgo/yoroi-frontend/issues/2448

**Comment by jarstorm** [comment]
Hello, and thank you for your contribution.
We're currently reviewing long-standing issues to keep the tracker manageable. As this issue hasn't seen activity in some time, we're closing it.
If the problem persists or you'd like to follow up, feel free to comment and we'll reopen it.
